### DISCUSSION TITLE
Cant get the data importer up for some reason

### DISCUSSION THREAD

**Discussion by JR2012thman** [firefly-iii/firefly-iii#8883] [Firefly III]
I have followed the guidelines of the documentation to get the data importer up and the docker container does provision and shows as healthy but I can't access it on port 81 or 8081. I tried in the .importer.env file to leave the url and vanity url blank, and fill it with my local network values, the documentation values, the internal values of the docker network. I am at a loss. What am I doing wrong and what can I provide to solve this issue. In the mean time I am just entering the transactions so that works but I would like to just see the web interface locally.

**Comment by JC5** [comment]
Very strange, does it give a white page or simply an error?
